gson/gson/src/main/java/com/google/gson/internal/bind/JsonTreeWriter.java

```diff
@@ -188,7 +188,6 @@ private void put(JsonElement value) {
       put(new JsonPrimitive(value));
       return this;
     }
```

Comment:
Needs to fix indentation to match formatting

Reply:
@jadshaker please address this in the next 2-3 days or we'll have to assign to someone else.

Reply:
Fixed